### DISCUSSION TITLE
Installation and running of wwatch3.v6.07

### DISCUSSION THREAD

**Discussion by Ikennaacmad** [NOAA-EMC/WW3#1207] [General]
Hi everyone,
Please I am new here as a user of wavewatch3 and believe that I will get the assistance I need to install wavewatch3 operationally.
**I need a step-by-step guide on how to install wwatch3.v6.07 and run it operationally.** 
So far, I have succeeded in downloading the wwatch3.v6.07.tar.gz, from the repository "https://github.com/NOAA-EMC/WW3/releases/tag/6.07".
I also executed but got stuck at the end of the following instruction: 
>>./install_ww3_tar
So any help to move forward would be appreciated.
Thank you in advance.

Ikenna

**Comment by JessicaMeixner-NOAA** [comment]
Here  is where you can find additional information: https://github.com/NOAA-EMC/WW3/wiki/Tech-Docs
